cms/djangoapps/contentstore/views/videos.py

```diff
@@ -31,6 +31,8 @@
     get_transcript_preferences,
     create_or_update_transcript_preferences,
     remove_transcript_preferences,
+    get_organization_transcript_credentials_state,
+    update_organization_transcript_credentials_state,
```

Comment:
@mushtaqak  These methods are renamed and their responses are also changed. You need to update accordingly.